ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +46,40 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+
+#  builder_run 'GUI-Tests' 1 ./test-gui.sh crud
+- artifact_dir: /tmp/artifacts
+  as: GUI-Tests
```

Comment:
I wouldn't use these names to avoid confusion with jenkins jobs. Maybe `crud`, `new-app`, `olm`, and `service-catalog`.

Reply:
Agree for long term, I'll update that soon.
And will remove the Jenkins jobs.

Reply:
OK, if we get this working and remove the Jenkins jobs, you'll want to remove the required status checks from

https://github.com/openshift/release/blob/1746b6149ebd92be7b33c8e6173cb0876089a7e9/cluster/ci/config/prow/config.yaml#L475-L488

Reply:
Got it, thanks!

Reply:
You can just remove those `required_status_checks` -- the required jobs that are managed by Prow get auto-detected FYI